modules/dnn/src/layers/einsum_layer.cpp

```diff
@@ -1312,92 +1311,41 @@ Mat LayerEinsumImpl::batchwiseMatMul(
     int K = input1ShapeOverride[2];
     int N = input2ShapeOverride[2];
 
-    std::vector<Mat> output;
+    Mat reshapedInput1 = input1;
+    Mat reshapedInput2 = input2;
```

Comment:
Could you elaborate why reshape is needed here? fastGemmBatch can already handle multiplification of n-dimensional matrices.

Reply:
This is required as some tensors might come in with dummy deletion BxNxMx1.

Reply:
Could you give a real Einsum example of `dummy deletion BxNxMx1`?

Reply:
@fengyuentau is there anything to change more? Could you please approve if not ?